iframe/style.css

```diff
@@ -1,14 +1,24 @@
+html {
+  font-size: 16px;
```

Comment:
This ensures `rem` values are always relative to a 16 pixel font size. While this is the default user agent font size for the `html` element in most browsers, in my experience, it's good to be explicit.